graphs/templates/base.html

```diff
@@ -109,7 +109,10 @@ <h4 class="modal-title" id="myModalLabel">Forgot Information</h4>
                     {% endif %}
                     <li><a href="{% url 'features' %}">Features</a></li>
                     <li><a href="{% url 'help_tutorial' %}">Help</a></li>
-                    <li><a href="{% url 'help_about' %}" style="text-align:center">About Us</a></li>                     
+                    <li><a href="{% url 'help_about' %}" style="text-align:center">About Us</a></li>  
+                    {% if uid != None %}
```

Comment:
![screen shot 2016-07-24 at 4 52 03 pm](https://cloud.githubusercontent.com/assets/5363860/17084472/01a469d4-51bf-11e6-8893-c04281b78c22.png)
The user will get a screen like this if its not a valid log in